src/main/java/ch/njol/skript/expressions/ExprDamagedItem.java

```diff
@@ -16,28 +19,37 @@
 import ch.njol.util.Kleenean;
 
 @Name("Damaged Item")
-@Description("Directly damages an item. In MC versions 1.12.2 and lower, this can be used to apply data values to items/blocks")
+@Description({"Directly changes the durability of an item. In MC versions 1.12.2 and lower, this can be used to apply data values to items/blocks",
+	"Damage is used to remove the specified number from the base durability of an item. E.g 400 - damage",
+	"Durability is used to modify the total durability"})
+
 @Examples({"give player diamond sword with damage value 100", "set player's tool to diamond hoe damaged by 250",
 	"give player diamond sword with damage 700 named \"BROKEN SWORD\"",
 	"set {_item} to diamond hoe with damage value 50 named \"SAD HOE\"",
-	"set target block of player to wool with data value 1", "set target block of player to potato plant with data value 7"})
+	"set target block of player to wool with data value 1", "set target block of player to potato plant with data value 7",
+	"give player wooden sword with 1 durability named \"VERY BROKEN SWORD\"",
+	"set player's tool to diamond hoe with durability 500"})
 @Since("2.4")
```

Comment:
Adding a new since `INSERT VERSION (with durability)` would help people know when it was added